### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by MDVAurelien**
Add some french translations

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -5,38 +5,38 @@
   " minute": "",
   " minutes": "",
   ".": "",
-  "0=Disabled, blank=inherit": "",
+  "0=Disabled, blank=inherit": "0=Désactivé, vide=hérité",
   "2FA": "",
   "<strong>Dataset:</strong> ": "",
-  "<strong>The following { n, plural, one {snapshot} other {# snapshots} } will be deleted. Are you sure you want to proceed?</strong>": "",
+  "<strong>The following { n, plural, one {snapshot} other {# snapshots} } will be deleted. Are you sure you want to proceed?</strong>": "<strong>Les { n, plural, one {snapshot} other {# snapshots} } seront supprimés. Êtes-vous sûr de vouloir continuer ?</strong>",
```
Comment (undsoft) [inline src/assets/i18n/fr.json:11]: You can translate the 'snapshot' and 'snapshots' too.
If French has different pluralization rules, you can adjust them. We are using ICU message format.

You can test here: http://format-message.github.io/icu-message-format-for-translators/editor.html
and press Learn more there for more examples.

Снимок экрана 2021-06-22 в 13 45 11

**Review by undsoft — APPROVED** [review]

**Comment by MDVAurelien** [comment]
Thks, very good !

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -5,38 +5,38 @@
   " minute": "",
   " minutes": "",
   ".": "",
-  "0=Disabled, blank=inherit": "",
+  "0=Disabled, blank=inherit": "0=Désactivé, vide=hérité",
   "2FA": "",
   "<strong>Dataset:</strong> ": "",
-  "<strong>The following { n, plural, one {snapshot} other {# snapshots} } will be deleted. Are you sure you want to proceed?</strong>": "",
+  "<strong>The following { n, plural, one {snapshot} other {# snapshots} } will be deleted. Are you sure you want to proceed?</strong>": "<strong>{ n, plural, one {L} other {Les} } { n, plural, one {'instantané} other {# instantanés} } { n, plural, one {sera} other {seront} } { n, plural, one {supprimé} other {# supprimés} }. Êtes-vous sûr de vouloir continuer ?</strong>",
```
Comment (undsoft) [inline src/assets/i18n/fr.json:11]: Looks like you wanted to do something like this:
```
<strong>{ n, plural, one {L 'instantané sera supprimé} other {Les # instantanés seront supprimés} }. Êtes-vous sûr de vouloir continuer ?</strong>
```

Comment (MDVAurelien) [reply src/assets/i18n/fr.json:11]: oh yes sorry i am not used to ICU Message Editor yet. Thank you that's it I'm correcting ;-)

**Pull request merged**